.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ commands:
         type: string
         default: ""
     steps:
-      - run: wget -qO - https://download.opensuse.org/repositories/Emulators:/Wine:/Debian/xUbuntu_18.04/Release.key | apt-key add -
+      - run: curl -fsSL https://download.opensuse.org/repositories/Emulators:/Wine:/Debian/xUbuntu_18.04/Release.key | gpg --dearmor | tee /etc/apt/trusted.gpg.d/wine.gpg > /dev/null
```

Comment:
Unrelated to this change, but how do we actually know this is the right key?

Reply:
Do you mean that we should check that the key is from opensuse? 


if there is a way to check that, we can add that to the code

if that can't be done, the only other way to do that would be to add the key as context in circleci and use it or (probably better option) create our own dockerfile with the key already present and use that as a base.

Reply:
> Do you mean that we should check that the key is from opensuse?

Yeah we'd need to compare the fingerprint since download.opensuse.org redirects to third-party mirrors that could be hosted by anyone.

>  the only other way to do that would be to add the key as context in circleci

This sounds like a good idea to me.